lib/sycamore/sycamore/tests/unit/transforms/test_merge_elements.py

```diff
@@ -239,6 +238,52 @@ def test_merge_elements(self):
         assert e.text_representation == ("Section-header1 on page 3")
         assert e.properties == {"page_number": 3, "columns": None, "page_numbers": [3], "rows": None, "title": None}
 
+    def test_merge_elements_image_summarize(self):
```

Comment:
how is this test different from previous?

Reply:
This is suppose to be with image summary, which I forgot to add while git committing.